### ISSUE TITLE
App Moves Entire UI to the Top After Editing A Note

### ISSUE THREAD

**Issue by SolodoloBest** [laurent22/joplin#7640]
.
-->

## Environment

Joplin version: 12.9.2
Platform: iPadOS
OS specifics: iPadOS16.2

## Steps to reproduce

1. Install Joplin app
2. Edit any note in the 'welcome' notebook
3. Watch the entire page shift up making the app unusable
![02FA3A39-355D-4AA9-86D4-B65447AF186F](https://user-images.githubusercontent.com/123171850/213697172-9c096aa2-e48f-44c9-9583-e746eb1f35e2.png)
![8BEA4304-6916-4EA9-B14B-35A12B438925](https://user-images.githubusercontent.com/123171850/213697177-da49c60e-1a80-409d-b3ad-d3c0b5d572bd.png)

**Comment by Daeraxa** [comment]
Can you enable the beta editor in the configuration and see if that resolves it?

**Comment by SolodoloBest** [comment]
Enabled beta-editor and then restarted the app. Bug persists. Even when making new notes 
![630FAAA7-DA16-45B3-90A3-B0A5968E2DA3](https://user-images.githubusercontent.com/123171850/213728159-b60c0630-ffcf-4b25-b2e2-59dc706a2182.png)

**Comment by laurent22** [comment]
Which version of iPad is it? It might help to know the screen dimensions in particular

**Comment by SolodoloBest** [comment]
iPad Pro (2018) 11 inches

**Comment by personalizedrefrigerator** [comment]
This looks like a duplicate of https://github.com/laurent22/joplin/issues/6512. As such, this should be fixed by https://github.com/laurent22/joplin/pull/7593, which should be in the next release.

For now, try disabling the floating keyboard.

**Comment by SolodoloBest** [comment]
Attached the Apple keyboard case to my iPad Pro which forces the on-screen keyboard to go away…

Bug **still** persists, except this time it goes away while I am actively creating or editing a note. The moment I leave the note, the bug appears. 

![IMG_4364](https://user-images.githubusercontent.com/123171850/213750171-81f5dc9e-f9ed-4568-b5c4-58be8d214efd.png)

![IMG_4365](https://user-images.githubusercontent.com/123171850/213750289-48aefdeb-2bf4-4840-a6d1-6c8c7b1a9a4c.png)

**Comment by personalizedrefrigerator** [comment]
The original issue had two causes:
1. The floating keyboard was enabled.
2. Crossfade transitions were enabled in accessibility settings (iOS settings > accessibility > motion > reduce motion > prefer cross-fade transitions)

The floating keyboard needs to be disabled before starting Joplin. To do this, open the floating keyboard and drag it to the bottom of the screen:
![Picture showing the floating keyboard with an arrow below it](https://user-images.githubusercontent.com/46334387/213766300-62fdece6-a3fd-4c89-8b4c-78b2705d5fad.png)

Hopefully this fixes the issue!

**Comment by laurent22** [comment]
I'm publishing a new beta version, so you might want to test if it fixes your issue: https://discourse.joplinapp.org/t/ios-pre-release-2-10-is-now-available-updated-29-dec/28848

**Comment by SolodoloBest** [comment]
> The original issue had two causes:
> 
> 1. The floating keyboard was enabled.
> 2. Crossfade transitions were enabled in accessibility settings (iOS settings > accessibility > motion > reduce motion > prefer cross-fade transitions)
> 
> The floating keyboard needs to be disabled before starting Joplin. To do this, open the floating keyboard and drag it to the bottom of the screen: ![Picture showing the floating keyboard with an arrow below it](https://user-images.githubusercontent.com/46334387/213766300-62fdece6-a3fd-4c89-8b4c-78b2705d5fad.png)
> 
![IMG_4366](https://user-images.githubusercontent.com/123171850/213778099-dfae2a7f-6efb-4d90-9728-debb885bc9cd.jpeg)

> Hopefully this fixes the issue!

I turned off ‘Floating Keyboard’ in Settings and ‘reduce motion’ has always been disabled. The bug still appears.
![IMG_4367](https://user-images.githubusercontent.com/123171850/213778135-cae5ba59-9f8b-4761-aafd-cce9988870d3.jpeg)
![IMG_4369](https://user-images.githubusercontent.com/123171850/213778254-3098476b-41cf-481b-8721-0b8b27888c2c.jpeg)

![IMG_4368](https://user-images.githubusercontent.com/123171850/213778272-c2eb2a3a-b5eb-4fa9-8492-8d153e97ede1.jpeg)

**Comment by SolodoloBest** [comment]
> I'm publishing a new beta version, so you might want to test if it fixes your issue: https://discourse.joplinapp.org/t/ios-pre-release-2-10-is-now-available-updated-29-dec/28848

I shared the screenshot with you on the TestFlight app but the bug is still present, sorry.
![IMG_4371](https://user-images.githubusercontent.com/123171850/213780112-9e702073-fc32-44c9-b2d8-1f5d105026e0.png)
![IMG_4370](https://user-images.githubusercontent.com/123171850/213780132-3286005c-c79f-4966-8405-ef6a161fdb30.jpeg)

**Comment by personalizedrefrigerator** [comment]
It can take a few days for new versions to show up on TestFlight. For me, it's still the version before the fix:
![](https://user-images.githubusercontent.com/46334387/213787715-cbb93d77-2559-45e0-9351-ee07d7781897.png)

Sorry for the confusion and thank you for your patience!

**Comment by SolodoloBest** [comment]
Thats the same version I have on TestFlight

**Comment by laurent22** [comment]
Yes to be clear it should be 12.10.2 but Apple didn't approve it yet

**Comment by SolodoloBest** [comment]
Great news! Bug Solved I removed the Apple Keyboard case and docked the Floating Keyboard. This stopped the bug from appearing, like you said @personalizedrefrigerator 


![IMG_4372](https://user-images.githubusercontent.com/123171850/213799269-bf210899-4983-405f-8118-837a1424988a.png)



@laurent22 should i close this ticket?



> The original issue had two causes:
> 
> 1. The floating keyboard was enabled.
> 2. Crossfade transitions were enabled in accessibility settings (iOS settings > accessibility > motion > reduce motion > prefer cross-fade transitions)
> 
> The floating keyboard needs to be disabled before starting Joplin. To do this, open the floating keyboard and drag it to the bottom of the screen: ![Picture showing the floating keyboard with an arrow below it](https://user-images.githubusercontent.com/46334387/213766300-62fdece6-a3fd-4c89-8b4c-78b2705d5fad.png)
> 
> Hopefully this fixes the issue!

**Comment by personalizedrefrigerator** [comment]
It looks like the new version is now available on TestFlight! If you upgrade, you should be able to check whether the floating keyboard-related issue is fixed.

**Comment by SolodoloBest** [comment]
Joplin 12.10.2 beta fixes the bug caused by floating Keyboard, closing ticket
